view/templates/head.tpl

```diff
@@ -44,6 +44,10 @@
 <script type="text/javascript" src="view/asset/perfect-scrollbar/js/perfect-scrollbar.jquery.min.js" ></script>
 <script type="text/javascript" src="view/js/acl.js" ></script>
 <script type="text/javascript" src="view/asset/base64/base64.min.js" ></script>
+<script type="text/javascript" src="view/js/addon-hooks.js" ></script>
+{{foreach $addon_hooks as $addon_hook}}
```

Comment:
What if `$addon_hooks` isn't set by any addon?

Reply:
`addon_hooks` is declared and initialised as an empty array in `addon-hooks.js`, so in that case no additional .js files will be added in `head.tpl`

Reply:
Double curly braces `{{ }}` denotes Smarty tags, and any variable mentioned between them are PHP template variables. This is completely independent from the Javascript variable.